endpoints/openrtb2/auction_test.go

```diff
@@ -3616,7 +3622,7 @@ var testStoredRequestsUuid = []string{
 		}
 	}`,
 	`{
-		"id": "ThisID",
+		"id": "{{UUID}}",
```

Comment:
With this change, there is no longer a test where we generate a uuid where we don't have the `{{UUID}}` macro. Shouldn't we still have a test that shows this case still works?